torch/csrc/jit/ir/ir.cpp

```diff
@@ -1825,6 +1826,19 @@ at::ArrayRef<Value*> createTupleUnpack(Value* v) {
 
 std::vector<Value*> inlineCallTo(Node* to_replace, Function* callee) {
   WithInsertPoint guard(to_replace);
+  if (!callee->isGraphFunction()) {
```

Comment:
I think the right place to edit is inliner.cpp not the call inlineCallTo. This should just fail for a builtin (since it is builtin it cannot be inlined).